zcutil/release-notes.py

```diff
@@ -0,0 +1,63 @@
+import re, sys, os, os.path
+import subprocess
+import argparse
+from itertools import islice
+
+# Returns dict of {'author': #_of_commits} from a release note
+def authors_in_release_notes(filename):
+    with open(dir + filename, 'r') as f:
+        authors = {}
+        line = f.readline()
+        commits = re.search('(\d+)', line).group(1)
+        first = re.sub(' \(\d+\)|:|\n', '', line)
+        authors[first] = commits
+        for line in f:
+            if line in ['\n', '\r\n']:
+                for author in islice(f, 1):
+                    commit_search = re.search('(\d+)', author)
+                    if commit_search:
+                        commits = commit_search.group(1)
+                    else:
+                        commits = 0
+                    author = re.sub(' \(\d+\)|:|\n', '', author)
+                    authors[author] = commits
+        return authors
+
+## Sums commits made by contributors in each Zcash release note in ./doc/release-notes and writes to authors.md
+def document_authors():
+    print "Writing contributors documented in release-notes directory to authors.md..."
+    with open('../doc/authors.md', 'w') as f:
+        f.write('Zcash Contributors\n==================\n\n')
+        total_contrib = {}
+        for notes in os.listdir(dir):
+            authors = authors_in_release_notes(notes)
+            for author in authors:
+                commits = int(authors[author])
+                if author in total_contrib:
```

Comment:
Well... because I knew them, basically. There's no way to get the handles from the local logs, and we probably shouldn't query the Github API from this script, so I'll just remove them. Maybe we can include the contributor Github ID's  in a wiki entry or somewhere else on Github.